src/views/vmd-home.view.scss

```
@@ -153,5 +153,8 @@
         &._bimedoc {
             max-height: 5.5rem;
         }
+        &._valwin {
+            max-height: 8rem;
```

J'aurais bien agrandi un peu le logo sur la home (qui me paraît un peu petit actuellement)
```suggestion
            max-height: 10rem;
```

Sur la page des résultats ça me paraît 👌